### ISSUE TITLE
Add x11 desktop environment and contribute to the project 

### ISSUE THREAD

**Issue by JesusChapman** [TermuxHacking000/SysO-Termux#4]
Hola amigo  es interesante tu proyecto, pero si me permites puedo hacerlo mejor.

¿Cómo?

1- añadir un entorno de escritorio
Termux actualmente está mejorando su entorno gráfico, un ejemplo de las GUI instalables en el:

https://github.com/Yisus7u7/termux-desktop-xfce

https://github.com/Yisus7u7/termux-desktop-lxqt

2- No solo eso, por lo que veo hay un gran desorden de carpetas, conciderando que ya la mayoría de herramientas que instala tu proyecto ya no tienen compatibilidad con termux.

3- cambiando la forma de hacer las cosas

Se crear paquetes .deb para termux, son paquetes instalables, podemos hacer un repositorio (el cual yo lo se hacer) de paquetes apt donde el usario solo ejecute : pkg install tool-name y instale el paquete fácilmente, hací no tendrá un reguero de carpetas, podrá actualizar las herramientas poniendo pkg update, y ahorrará espacio, también habrá más facilidad de uso

Pd: si no sabes como luce el escritorio de termux con x11-repo, es hací :  

![Screenshot_20210531-103533](https://user-images.githubusercontent.com/64093255/120575499-ccc09680-c3e6-11eb-8450-2647e74431fd.png)

**Comment by churrito123** [comment]
'-'

**Comment by JesusChapman** [comment]
Que succede?

**Comment by churrito123** [comment]
> Que succede?

Iba a mostrar exactamente lo mismo, pero parece que fuiste más rápido.

**Comment by JesusChapman** [comment]
Oh ok

**Comment by JesusChapman** [comment]
Pero exatamente lo mismo? ;-;

**Comment by churrito123** [comment]
si hombre, de extraño que parezca

**Comment by JesusChapman** [comment]
Ok tu podrias colaborar tambien

**Comment by churrito123** [comment]
> Ok tu podrias colaborar tambien

Cooperaré más, llevará un tiempo porque cometí un error que me hizo perder mi cuenta principal de github.

**Comment by JesusChapman** [comment]
:v, vale, e compilado java (jdk) para termux de 32 bits

**Comment by churrito123** [comment]
> :v, vale, e compilado java (jdk) para termux de 32 bits

Ok

**Comment by 0n1cOn3** [comment]
Suena como un montón.

Lo que sí podría querer saber son los paquetes que ya no parecen funcionar con Termux (o los paquetes en general de SysO-Termux) para sustituirlos o eliminarlos si es necesario.

Saludos desde Suiza

**Comment by JesusChapman** [comment]
Igual, yo voto más por hacer paquetes deb, si no saben que es un repositorio con paquetes deb, aquí hay un ejemplo :  

https://github.com/Yisus7u7/yisus.ports

**Comment by 0n1cOn3** [comment]
> Igual, yo voto más por hacer paquetes deb, si no saben que es un repositorio con paquetes deb, aquí hay un ejemplo :  
> 
> https://github.com/Yisus7u7/yisus.ports 

Ya creé paquetes deb para Cydia (iOS Jailbreak Store) hace 10 años ;-) Estoy muy familiarizado con la creación de paquetes deb :).

Sólo que, ¿por qué un repo separado?

**Comment by JesusChapman** [comment]
Que querés decir?

**Comment by JesusChapman** [comment]
Bueno, al parecer @Darkmux no contesta, iniciaré entonces yo mismo el proyecto por separado y lo marcaré como un fork

**Comment by JesusChapman** [comment]
![Screenshot_20210603-090112](https://user-images.githubusercontent.com/64093255/120721894-5ed2a880-c494-11eb-9f60-71342a810a41.png)
![Screenshot_20210603-090631](https://user-images.githubusercontent.com/64093255/120721897-609c6c00-c494-11eb-8fe3-fefe3b0b5f92.png)
![Screenshot_20210603-090525](https://user-images.githubusercontent.com/64093255/120721899-61350280-c494-11eb-8658-059c3c4c1aa9.png)

Bueno, iniciado el proyecto por mi cuenta

**Comment by Darkmux** [comment]
Hola amigos, agradezco su apoyo, sin embargo, no le doy mantenimiento a
SysO-Termux, ya que solo trabajo en Dark-Termux...

**Comment by JesusChapman** [comment]
Hmm, entonces haré el fork de proyecto

**Comment by JesusChapman** [comment]
Pero con diferente nombre

**Comment by 0n1cOn3** [comment]
> Hmm, entonces haré el fork de proyecto 

Ya lo he bifurcado. Mis cambios hasta ahora sólo se refieren a la traducción al inglés.

En caso contrario, se puede continuar con este proyecto. 😄

Give a look: https://github.com/0n1cOn3/SysO-Termux

**Comment by JesusChapman** [comment]
Yo también ya lo hize, pero uso el sistema de paqueteria apt, el usario solo debe ejecutar : 

`pkg install tool-name`

Y listo, instalará la herramienta, me falta pulir un poco el README

https://github.com/Yisus7u7/SysO-Termux

**Comment by 0n1cOn3** [comment]
> Que querés decir? 

A eso me refería jeje. 
No tengo claro por qué tienes que crear tu propio repo, mientras no hayas creado tus propias aplicaciones / scripts. Por lo demás, nada.

**Comment by JesusChapman** [comment]
Cree el mismo propio para poder usar usar GitHub pages

**Comment by JesusChapman** [comment]
![Screenshot_20210603-211355](https://user-images.githubusercontent.com/64093255/120737361-8edd7400-c4b3-11eb-8c69-ccd0ea630261.png)

Funciona a la perfección

**Comment by 0n1cOn3** [comment]
> Yo también ya lo hize, pero uso el sistema de paqueteria apt, el usario solo debe ejecutar : 
> 
> 
> 
> `pkg install tool-name`
> 
> 
> 
> Y listo, instalará la herramienta, me falta pulir un poco el README
> 
> 
> 
> https://github.com/Yisus7u7/SysO-Termux

Soy amigo de Git :3

**Comment by JesusChapman** [comment]
UwU, aun haci prefiero usar un sistema de paqueteria apt para el usuario

**Comment by 0n1cOn3** [comment]
También podríamos ampliarlo hasta el punto de tener hasta cuatro idiomas. Incluye actualizaciones de herramientas, etc. 

Actualmente uso deepl, porque no sé español. Pero también alemán, un poco de francés, algunas palabras de italiano y, por supuesto, inglés.

**Comment by 0n1cOn3** [comment]
> UwU, aun haci prefiero usar un sistema de paqueteria apt para el usuario 

Pero hay que empaquetar todo una y otra vez, con una actualización. Sin runner/script que se vuelve tedioso con el número de paquetes.

Edit:
Con los scripts, sincroniza una vez en Github. Git pull y todo está actualizado

**Comment by JesusChapman** [comment]
No, es más fácil con debs, y no se si lo sabias, hay algo llamado  postinst  y postrm

Son scripts que se ejecutan luego de la instalación de un paquete deb

Haci que solo sería cambiar ese script y subir la versión del deb para que se actualize

**Comment by JesusChapman** [comment]
Pd si el problema es que a usted se le dificulta la creación de paquetes deb, eso lo entiendo, por eso creé un entorno de compilación para hacerlos fácilmente :  

https://github.com/Yisus7u7/termux-builder

**Comment by 0n1cOn3** [comment]
Nunca entendí eso con lo de los puestos.

¡Qué gran cosa se puede hacer! Le echaré un vistazo.

**Comment by JesusChapman** [comment]
Vale bro

**Comment by 0n1cOn3** [comment]
> https://github.com/Yisus7u7/termux-builder

Ah, sí... Ahora veo lo que no entendí 🤣 Oh hombre, es simple 🤦🏻‍♂️

**Comment by JesusChapman** [comment]
Demasiado simple, y si miras mi repositorio

https://github.com/Yisus7u7/SysO-Termux

Te darás cuenta lo simple que es

**Comment by 0n1cOn3** [comment]
> Demasiado simple, y si miras mi repositorio
> 
> https://github.com/Yisus7u7/SysO-Termux
>
> Te darás cuenta lo simple que es 

Si 🥲

**Comment by JesusChapman** [comment]
Jajajajaja, simple y a la vez  moderno

**Comment by JesusChapman** [comment]
![Screenshot_20210604-130407](https://user-images.githubusercontent.com/64093255/120844756-7ca41a00-c535-11eb-8770-4fc399819a80.png)
Ya e logrado compilar exitosamente varios paquetes

**Comment by JesusChapman** [comment]
Como van._?

**Comment by JesusChapman** [comment]
Eh cancelado mi lo de este fork, mi proyecto de paquetes para termux ahora es algo mas robusto y complejo, e tenido que crear una página para el :


https://pkgs-termux.github.io/pkgs.org


Ya está disponible una gran cantidad de paquetes

**Comment by 0n1cOn3** [comment]
En los sentidos, la bifurcación sigue existiendo, sólo que de forma diferente 👍🏻🤗

**Comment by JesusChapman** [comment]
No tiene tantas cosas de hacking, como ve, hay muchos temas y eh compilado lib wayland-protocols para termux

**Comment by 0n1cOn3** [comment]
Ya veo, ya veo. 

Oh, Wayland bajo Termux 👀 wow.

**Comment by JesusChapman** [comment]
Si, mire esto:

**Comment by Yisus7u7** [comment]
Si, mire esto:

**Comment by JesusChapman** [comment]
![Screenshot_20210707-212924](https://user-images.githubusercontent.com/64093255/125018796-6b21c680-e03b-11eb-81df-e7d487eff4c9.png)
![Screenshot_20210707-212837](https://user-images.githubusercontent.com/64093255/125018830-7ecd2d00-e03b-11eb-951e-738bc2f07d58.png)
![Screenshot_20210708-141629](https://user-images.githubusercontent.com/64093255/125018833-7f65c380-e03b-11eb-88b5-59686e14ffd1.png)
![Screenshot_20210707-213128](https://user-images.githubusercontent.com/64093255/125018835-7ffe5a00-e03b-11eb-9573-f7ea350ce28b.png)
![Screenshot_20210707-212934](https://user-images.githubusercontent.com/64093255/125018837-8096f080-e03b-11eb-9c25-4a93496a9a6c.png)
![Screenshot_20210707-213615](https://user-images.githubusercontent.com/64093255/125018838-8096f080-e03b-11eb-8828-9eec0be695f1.png)
![Screenshot_20210707-213546](https://user-images.githubusercontent.com/64093255/125018840-812f8700-e03b-11eb-9031-3ff42c8c0ba6.png)
![Screenshot_20210707-214421](https://user-images.githubusercontent.com/64093255/125018849-82f94a80-e03b-11eb-83c9-72c65c0537aa.png)
![Screenshot_20210707-213422](https://user-images.githubusercontent.com/64093255/125018851-82f94a80-e03b-11eb-8222-5708a4562043.png)
![Screenshot_20210707-213353](https://user-images.githubusercontent.com/64093255/125018852-8391e100-e03b-11eb-8528-93c4b91fe89b.png)
![Screenshot_20210707-213149](https://user-images.githubusercontent.com/64093255/125018853-842a7780-e03b-11eb-87f9-5b3b35616d52.png)
![Screenshot_20210708-145445](https://user-images.githubusercontent.com/64093255/125018854-84c30e00-e03b-11eb-9d1b-846eb80bbacb.png)
@0n1cOn3
@Darkmux

**Comment by 0n1cOn3** [comment]
Amazing!

**Comment by JesusChapman** [comment]
Yes!, an pasado muchas  cosas desde que abrí esta discusión aquí XD
